WorkflowGuidelines.md

```diff
@@ -79,6 +79,32 @@
       let savedData = System.IO.File.ReadAllText saveFilePath.FullName
       ```
 
+    * Discarding exceptions:
+
+        Discarding exceptions is a bad practice because it can lead to unexpected behavior and bugs. It's better to handle the exception in some way, or at least log them to help with debugging if something unex
+    
+        Example:
+        ```f#
+        let func zipPath =
+            try
+                ZipFile.Open(zipPath.ToString(), ZipArchiveMode.Read) |> ignore
+            with
+            | ex -> 
+                ()
+        ```
+        ```f#
```

Comment:
@parhamsaremi are you sure this syntax-lang signaling works? I thought the keyword would be "fsharp", not "f#"

Reply:
It was highlighted in vscode so I thought it's right. I'll check this